concepts/evaluation/queue.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: "Annotation Queues"
+---
+
+The Literal AI platform helps you organise and smoothen the human evaluation process by easily setting-up annotation queues.
+Each queue let you add data for review and provides an accessible and efficient ui for domain experts to score, tag and add to datasets crucial insights.
+
+## Creating an annotation queue
+
+You can access your queues on the "Annotation Queues" page, and create them by pressing the "+" cta in the top right corner of the table.
```

Comment:
IMO better to avoid jargon in public documentation.

```suggestion
You can access your queues on the "Annotation Queues" page, and create them by pressing the "+" button in the top right corner of the table.
```